independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json

```diff
@@ -0,0 +1,55 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "UK Bank Holidays",
+    "description": "Bank holidays list retrieved from Government web site https://www.gov.uk",
+    "version": "1.0",
+    "contact": {
+      "name": "Martyn Lesbirel",
+      "email": "ipc-author-bh@outlook.com"
```

Comment:
@martynlesbirel Can you please provide `website` property.

Documentation - [Adding metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
Website added

Reply:
@martynlesbirel The `url` property is not showing up